demo/csi/aws/ebs/example-disk.tf

```diff
@@ -0,0 +1,30 @@
+data "aws_kms_alias" "ebs" {
+  name = "alias/aws/ebs"
```

Comment:
There's no other reference to setting up a KMS key in the documentation. Can we either add something giving folks a pointer here or remove this?

Reply:
AWS automatically provides a master key for EBS to use to decrypt the per-volume keys, which is what is being used here. There is nothing extra to set up in this case. You can also use KMS to create other master keys, which requires more setup. I have added a comment to this effect.